docs/pages/deploy-a-cluster/deployments/aws-active-passive-deployment.mdx

```diff
@@ -0,0 +1,111 @@
+  [Failover Routing Policy](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-failover-types.html#dns-failover-types-active-passive) 
+  with the active cluster loadbalancer as the primary record and the passive cluster loadbalancer as the secondary record.
+  - Use health checks to determine Teleport cluster availability. Route53 will automatically send the traffic to the 
+    passive cluster during the event of a regional outage and it will automatically routes the traffic back to the active 
+    cluster once the Active cluster when health checks report a normal state.
+
+### Advantages of this deployment architecture
+
+- Having a cluster in multi-regions makes the clusters quickly available during the event of regional outages.
+- Downtime can be minimized vs traditional disaster recovery from a backup.
+- All required Teleport components can be provisioned within the AWS ecosystem.
+- High-availability Auto Scaling group of Auth Service pods that must remain in a Primary region
+- High-availability Auto Scaling group of Proxy Service pods deployed across multiple regions
+
+### Disadvantages of this deployment architecture
+
+- Long-term cost may be a prohibitive factor for some organizations and can increase total cost of ownership
+  (TCO) throughout the system's lifetime cycle.
+- Deploying and maintaining the added layer of regional components takes more engineering effort.
+- More technically complex to deploy than a single region Teleport cluster or simple disaster recovery.
+
+### Configuration Example with Teleport Kubernetes and Helm:
+
+##Prerequisites
+
+(!docs/pages/kubernetes-access/helm/includes/teleport-cluster-prereqs.mdx!)
+
+## Step 1/6. Install Helm
+
+(!docs/pages/kubernetes-access/helm/includes/teleport-cluster-install.mdx!)
+
+## Step 2/6. Add the Teleport Helm chart repository
+
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+## Step 3/6. Set up Teleport Cluster in Primary Region (In this example us-west-1 is the Primary Region):
+- Running an HA Teleport cluster using [AWS, EKS, Helm](https://goteleport.com/docs/deploy-a-cluster/helm-deployments/aws/)
+- Configure [Single Sing-on](https://goteleport.com/docs/access-controls/sso/) and test your access to the Teleport cluster.
+
+## Step 4/6. Setup DynamoDB & S3 Global replication to Secondary Region (In this example us-east-1 is the Secondary Region):
+- Check the Key Components section of this document for DynamoDB Global replication and S3 Replication.
+- Post successful completion of Replication DynamoDB and S3 proceed to Step 5/6.
+
+## Step 5/6. Set up Teleport cluster in Secondary Region same as Step 3/6:
+- We will use  same (Primary Region cluster) `aws-values.yaml` & `aws-issuer.yaml` config files to install Teleport name space in Secondary Region (us-east-1) by changing the clusterName.
+  So that it will use same backend as defined in Primary Region (us-west-1) for DynamoDB and S3 buckets. 
```

Comment:
If we do this, wouldn't a regional outage in `us-west-1` prevent the `us-east-1` cluster from being able to function? I think we should use a separate values file which targets the DynamoDB tables in the `us-east-1` region instead.

Also, we should provide full examples of the values files to use in this guide.

Reply:
No, if we enable the kube operator on us-east-1 region, it will function normally, I did tested in my test bed.
I will update the aws-values.yaml for both regions.